docs/en_US/query_tool_toolbar.rst

```diff
@@ -214,6 +214,37 @@ Data Editing Options
    | SQL                  | Use the SQL button to check the current query that gave the data.                                 |                |
    +----------------------+---------------------------------------------------------------------------------------------------+----------------+
 
+
+Pagination Options
+********************
+
+.. image:: images/query_data_pagination.png
+    :alt: Query tool data pagination options
+    :align: center
+
+.. table::
+   :class: longtable
+   :widths: 1 4 1
+
+   +----------------------+---------------------------------------------------------------------------------------------------+----------------+
+   | Icon                 | Behavior                                                                                          | Shortcut       |
+   +======================+===================================================================================================+================+
+   | *Page Range*         | Show the current row numbers visible in the data grid.                                            |                |
+   +----------------------+---------------------------------------------------------------------------------------------------+----------------+
+   | *Edit Page Range*    | Click to open the from and to page range inputs to allow setting them.                            |                |
+   +----------------------+---------------------------------------------------------------------------------------------------+----------------+
```

Comment:
Change this documentation as we are going to rename it to "Showing Rows:" and "Edit range"